ext/active_span_source.py

```diff
@@ -49,3 +53,21 @@ def active_span(self, loop=None):
         # get the current running Task for this loop and return the ActiveSpan
         task = asyncio.Task.current_task(loop=loop)
         return getattr(task, '__active_span', None)
+
+    def deactivate(self, span, loop=None):
```

Comment:
Yes that was my feeling too. Having it here means that the `ActiveSpanSource` knows how to handle all the context propagation, so for setting, getting and deactivating the `Span`. I think that for the Java API we don't have any other option than the one it was proposed.